lib/model/content.dart

```diff
@@ -1093,10 +1109,32 @@ class _ZulipContentParser {
     } else {
       return UnimplementedBlockContentNode(htmlNode: divElement);
     }
+
+    double? originalWidth, originalHeight;
+    final originalDimensions = imgElement.attributes['data-original-dimensions'];
+    if (originalDimensions != null) {
+      // Server encodes this string as "{width}x{height}" (eg. "300x400")
+      final cutsets = originalDimensions.split('x');
```

Comment:
As part of my usual performance paranoia for the content parser, let's avoid `split` and allocating a list here: instead can find the index of the first 'x' and parse the substrings before and after (if there's a second `x`, it'll cause the int parsing to fail), or can use a regexp like `^(\d+)x(\d+)$`.